Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -3,6 +3,22 @@ cmake_minimum_required (VERSION 3.12)
 # ITK --stop ]]
 project (HDF5 C)
 
+if (POLICY CMP0074)
```

Comment:
This was removed in the previous commit. Was the introduction intentional?

Reply:
Minimizing diffs.  Not needed for ITK's minimum cmake version, but is needed for HDF5's minimum version.

I decided to keep it in for now.